anyway/flask_app.py

```diff
@@ -131,8 +130,8 @@ def generate_csv(results):
             output.writeheader()
 
         row = {k: v.encode('utf8')
-               if type(v) is six.text_type else v
-               for k, v in iteritems(serialized)}
+               if type(v) is str else v
```

Comment:
That's weird (the original test, too). 
```suggestion
               if isinstance(v, str) else v
```

Reply:
I have a few other fixes I want to add to this file, but as you suggested, I'll add them in a separate PR.